dts/bindings/dac/atmel,sam0-dac.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2020 Google LLC.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Atmel SAM0 family DAC
+
+compatible: "atmel,sam0-dac"
+
+include: dac-controller.yaml
+
+properties:
+    reg:
+      required: true
+
+    clocks:
+      required: true
+
+    clock-names:
+      required: true
+
+    label:
+      required: true
+
+    reference:
+      type: string
+      required: false
+      description: Reference voltage source
+      enum:
+        - "int1v"
+        - "avcc"
+        - "vrefp"
+
+    "#io-channel-cells":
+      const: 1
+
+io-channel-cells:
+    - output
```

Comment:
As this DAC does not have different channels and the output always goes to pin PA02, I think the `output` cell can be removed and `#io-channel-cells` set to 0.

Reply:
Done.  I hope to add SAMD51 support in the future which does have two channels.